_posts/2019-10-18-parent-instance-error.md

```diff
@@ -0,0 +1,16 @@
+---
+site: freiburg
+tags: [devops, bug]
+title: Parent instance error
+location: Freiburg, Germany
+---
+
+UseGalaxy.eu is currently facing some issues with tools execution and it may take us a while to find a solution.  
```

Comment:
There is a PR on its way ... if we are brave we can/should try it. https://github.com/galaxyproject/galaxy/pull/8818

Reply:
I am monitoring the PR waiting for the end of tests

Reply:
I think that PR is ready for merge, please let me know if you are planning to do some testing and I'll wait to press the button.

Reply:
Yes, I am going to test it right now.